spec/services/tax_tribs/status_spec.rb

```diff
@@ -34,17 +34,36 @@
   end
 
   describe '.version' do
+    let(:git_result) { spy('git_result') }
```

Comment:
I'd prefer `double('git_result')` (I don't think you'd need the implicit `.as_null_object` of the `spy`, here).  You're not making any `have_received` calls, which the use of `spy` implies...